docker-compose.production.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.2"
+services:
+ db:
+   image: mysql:5.7
+   env_file:
+     - '.env'
+   ports:
+     - "3306:3306"
+   command: ['mysqld', '--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+   healthcheck:
+     test: ["CMD", "mysqladmin", "ping", "-h", "localhost","-plinks"]
+     timeout: 20s
+     interval: 10s
+     retries: 10
+ migrate:
+   image: wikipedialibrary/externallinks:latest
+   entrypoint: ["python", "/app/manage.py", "migrate"]
+   restart: on-failure
+   env_file:
+     - '.env'
+   depends_on:
+     - 'db'
+   volumes:
+     - ./:/app
```

Comment:
externallinks performs migrations on startup. we should either remove the migration step from its `gunicorn.sh` script and just have this service, or remove this service.

Reply:
It should have a build target.

Reply:
Gotcha! I made the changes requested in the `production` and `staging` docker-compose files. I pretty much had copied the production file from the staging file in the server (that is also how it's setup in `wikilinks-prod`). When we pull these changes to staging and both production instances, what do we have to do to run the new setup? Do we just build the image and run it?

Reply:
what we're doing with these docker-compose.environment.yml files is defining deviations from docker-compose.yml. We'd then go to a fresh server with docker compose and docker swarm and deploy the stack, something like
`docker stack deploy -c docker-compose.yml -c docker-compose.staging.yml staging`
which would deploy the stack with a merged configuration of those two files.

An issue we have here is that by the time we're done eliminating all of the extraneous things in docker-compose.staging.yml and docker-compose.production.yml, there's nothing left. We don't need them. We could just drop those files altogether and the stack deployment would be `docker stack deploy -c docker-compose.yml stackname`

Reply:
Awesome! I guess I will remove those files and rename and restructure my PR so that it actually reflects the changes made.